includes/admin/class-llms-admin-header.php

```diff
@@ -68,7 +69,7 @@ public function admin_header() {
 		// phpcs:enable WordPress.Security.ValidatedSanitizedInput.MissingUnslash
 
 		// Exclude the wizard.
-		if ( ! empty( $_GET['page'] ) && 'llms-setup' === $_GET['page'] ) {
+		if ( ! empty( $_GET['page'] ) && in_array( $_GET['page'], array( 'llms-setup', 'llms-cohorts') ) ) {
```

Comment:
```suggestion
		if ( ! empty( $_GET['page'] ) && in_array( $_GET['page'], array( 'llms-setup', 'llms-cohorts', true ) ) ) {
```
This is to fix the phpcs, but actually I wouldn't do it this way. I would rather add a filter hook that any third party hook can leverage

Reply:
Added `llms_admin_show_header` filter